cluster/addons/dns/skydns-rc.yaml.in

```diff
@@ -21,15 +21,17 @@ desiredState:
         dnsPolicy: "Default"  # Don't use cluster DNS.
         containers:
           - name: etcd
-            image: quay.io/coreos/etcd:v2.0.3
+            image: gcr.io/google_containers/etcd:2.0.9
```

Comment:
As long as we have a Dockerfile for it, I am ok.  I see that it is an ultra-minimal Dockerfile, which I like.  But it still pulls code from externally.  We should probably vendor it in, no?

@dchen1107